velero-plugin-for-aws/object_store.go

```diff
@@ -342,6 +395,11 @@ func (o *ObjectStore) GetObject(bucket, key string) (io.ReadCloser, error) {
 		Key:    &key,
 	}
 
+	if o.sseCustomerKey != "" {
+		req.SSECustomerAlgorithm = aws.String("AES256")
+		req.SSECustomerKey = &o.sseCustomerKey
+	}
```

Comment:
[1]

Reply:
https://github.com/vmware-tanzu/velero-plugin-for-aws/pull/96#discussion_r803673757